code/modules/reagents/oldchem/reagents/reagents_toxin.dm

```diff
@@ -26,6 +26,234 @@
+			alternate_ending = 1
+		to_chat(M,"<span class='danger'> The shapes extend tendrils out of your wound... no... those are legs! SPIDER LEGS! You have spiderlings growing inside you! You scratch at the wound, but it just aggrivates them - they swarm out of the wound, biting you all over!</span>")
+		M.visible_message("<span class='danger'>[M] flails around on the floor as spiderlings erupt from their skin and swarm all over them! </span>")
+		M.Stun(20)
+		M.Weaken(20)
+		// yes, this is a hella long stun - that's intentional. Gotta give the spiderlings time to escape.
+		var/obj/effect/spider/terror_spiderling/S1 = new(get_turf(M))
+		S1.grow_as = /mob/living/simple_animal/hostile/poison/terror_spider/red
+		S1.name = "red spiderling"
+		var/obj/effect/spider/terror_spiderling/S2 = new(get_turf(M))
+		S2.grow_as = /mob/living/simple_animal/hostile/poison/terror_spider/gray
+		S2.name = "black spiderling"
+		var/obj/effect/spider/terror_spiderling/S3 = new(get_turf(M))
+		S3.grow_as = /mob/living/simple_animal/hostile/poison/terror_spider/green
+		S3.name = "green spiderling"
+		M.adjustBruteLoss(20)
+		M.adjustToxLoss(40)
+		if (alternate_ending)
+			// This is the alternate ending of the infestation, triggered above by someone bringing back an infection from a gateway mission to the main station.
+			// In this ending, we still spawn spiderlings - but we make sure they're stillborn, and don't grow up.
+			// In addition, we give you an extra 60 toxin... enough to crit most people.
+			S1.stillborn = 1
+			S2.stillborn = 1
+			S3.stillborn = 1
+			M.adjustToxLoss(60)
+	if (wdstage == 190) // 6m 30s
+		to_chat(M,"<span class='danger'>The spiderlings are gone. Your wound, though, looks worse than ever. Remnants of tiny spider eggs, and dead spiders, inside your flesh. Disgusting.</span>")
+		M.reagents.remove_reagent("terror_white_toxin", 100)
+	..()
+	return
+
+// Terror Spider, White, Anti Toxin
+
+/datum/reagent/terror_white_antitoxin
+	name = "White Spider Antitoxin"
+	id = "terror_white_antitoxin"
+	description = "A strange serum that destroys spider eggs."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.1
+
+
+// Terror Spider, Queen
+
+/datum/reagent/terror_queen_toxin
+	name = "Terror Queen venom"
+	id = "terror_queen_toxin"
+	description = "A royally potent venom."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 2
+
+/datum/reagent/terror_queen_toxin/on_mob_life(var/mob/living/M as mob)
+	if(!M) M = holder.my_atom
+	// make them hallucinate a lot, like a changeling sting
+	if (M.hallucination < 400)
+		M.hallucination += 50
+	return
+
+
+// Terror Spider, Empress
+
+/datum/reagent/terror_empress_toxin
+	name = "Terror Empress venom"
+	id = "terror_empress_toxin"
+	description = "Distilled evil."
+	reagent_state = LIQUID
+	color = "#CF3600" // rgb: 207, 54, 0
+	metabolization_rate = 0.5 // like ketamine, but lasts longer.
```

Comment:
why not just make it a a child of ketamine then, with a different metabolization rate...or just have it inject a higher amount of ketamine?